src/lib/util/logger/Logger.ts

```diff
@@ -0,0 +1,197 @@
+import { Util } from '../Util';
```

Comment:
I think you used code from is-logger right? Can you put a link at the top to the original project? (only if you used code)